lib/interface_quda.cpp

```diff
@@ -185,6 +189,15 @@ static TimeProfile profileGaugeObs("gaugeObservablesQuda");
 //!< Profiler for gaugeSmearQuda
 static TimeProfile profileGaugeSmear("gaugeSmearQuda");
 
+//!< Profiler for APEQuda
```

Comment:
I think these can be removed, they look residual of some code that's been deleted.

Reply:
removed in 2ace7a2a010643023722cfc4c4b9758ea85e7073